modules/nw-gcp-installing-global-access-configuration.adoc

```diff
@@ -25,6 +25,15 @@ $ ./openshift-install create manifests --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the name of the directory that
 contains the `install-config.yaml` file for your cluster.
 +
+. Create a file that is named `cluster-ingress-default-ingresscontroller.yaml` in the `<installation_directory>/manifests/` directory:
++
+[source,terminal]
+----
+$ touch <installation_directory>/manifests/cluster-ingress-default-ingresscontroller.yaml <1>
```

Comment:
As it's a newly created YAML file, the subsequent section `3. Open the cluster-ingress-default-ingresscontroller.yaml file in an editor...` should tell the complete content of a sample YAML file instead, for example, 
```
apiVersion: operator.openshift.io/v1
kind: IngressController
metadata:
  creationTimestamp: null
  name: default
  namespace: openshift-ingress-operator
spec:
  endpointPublishingStrategy:
    loadBalancer:
      providerParameters:
        gcp:
          clientAccess: Global 
        type: GCP
      scope: Internal          
    type: LoadBalancerService

Reply:
@gpei Could you help review the content of the sample YAML too, I'm not quite sure it is correct, thanks!

Reply:
@jianli-wei seen from our flexy repo, the YAML content is correct.

`Changing the default ingress LoadBalancer with Global access enabled on GCP`
```
 cat <<EOF >manifests/cluster-ingress-default-ingresscontroller.yaml
apiVersion: operator.openshift.io/v1
kind: IngressController
metadata:
  name: default
  namespace: openshift-ingress-operator
spec:
  endpointPublishingStrategy:
    loadBalancer:
      scope: Internal
      providerParameters:
        type: GCP
        gcp:
          clientAccess: Global
    type: LoadBalancerService
EOF
```